Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -26,14 +27,15 @@ import Mathlib.RingTheory.HahnSeries.Summable
 * Basic properties of Hasse derivatives
 
 -/
+universe u
 
 open scoped Classical
 open HahnSeries BigOperators Polynomial
 
 noncomputable section
 
 /-- A `LaurentSeries` is implemented as a `HahnSeries` with value group `ℤ`. -/
-abbrev LaurentSeries (R : Type*) [Zero R] :=
+abbrev LaurentSeries (R : Type u) [Zero R] :=
```

Comment:
Do we get a substantial benefit from adding an explicit universe variable here?

Reply:
I don't think we get, it seems more coherent with the choice made in all files concerning Rational Functions. But I am of course very happy to revert it if you suggest so.

Reply:
There is no need to revert.  I just wanted to know if there was an interesting reason.